release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -694,6 +694,26 @@ In the following tables, features are marked with the following statuses:
 
```

Comment:
Beyond the scope of this PR, but the empty table is a little odd. Assuming that something will go here eventually. :)